Gopkg.toml

```diff
@@ -81,3 +81,6 @@ version="v1.4.7"
 [[override]]
   name = "github.com/tensorflow/tensorflow"
   version = "=v1.13.1"
+
+[[constraint]]
+  name = "github.com/google/go-cmp"
```

Comment:
Can we specify the version ? Also seems it does not prune non-go and unused packages.

Reply:
Specified version